visidata/unfurl.py

```diff
@@ -30,7 +31,15 @@ def reload(self):
         self.rows = []
         unfurl_empty = self.options.unfurl_empty
         for row in Progress(self.source.rows):
-            val = self.source_col.getValue(row)
+            try:
+                val = self.source_col.getValue(row)
+            except Exception as e:
+                e.stacktrace = stacktrace()
+                if unfurl_empty:
+                    self.addRow([row, TypedExceptionWrapper(None, exception=e), TypedExceptionWrapper(None, exception=e)])
```

Comment:
UnfurlSheet needs a rowdef at top of class (not your fault obviously but I need to figure it out to best review this line :).